2024-03-15T09:20:47 {"cpu": 0.54931640625, "cpu_total": 2, "cpu_usage": 0.274658203125, "memory": 2833.0, "memory_total": 7929.0, "memory_usage": 0.35729600201790895, "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "Pipeline export_to_big_query for run 3 in schedule 2 is alive.", "timestamp": 1710494447.473922, "uuid": "fc99fb9dabf64657b1fe7194dde1d626"}
2024-03-15T09:20:47 {"block_run_id": 17, "block_uuid": "load_customer_dimension", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "Execute PipelineRun 3, BlockRun 17: pipeline export_to_big_query block load_customer_dimension", "timestamp": 1710494447.917457, "uuid": "98401590956343559dadfec0efba40fc"}
2024-03-15T09:20:48 {"cpu": 0.54931640625, "cpu_total": 2, "cpu_usage": 0.274658203125, "memory": 2891.0, "memory_total": 7929.0, "memory_usage": 0.36461092193214784, "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "Pipeline export_to_big_query for run 3 in schedule 2 is alive.", "timestamp": 1710494448.324446, "uuid": "a489fa02e3bb432b894de072cc7a108f"}
2024-03-15T09:20:49 {"block_run_id": 19, "block_uuid": "load_department_dimension", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "Execute PipelineRun 3, BlockRun 19: pipeline export_to_big_query block load_department_dimension", "timestamp": 1710494449.02402, "uuid": "c8ede126072440af9863a2ad6251166e"}
2024-03-15T09:20:49 {"block_run_id": 24, "block_uuid": "load_metadata_dimensio", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "Execute PipelineRun 3, BlockRun 24: pipeline export_to_big_query block load_metadata_dimensio", "timestamp": 1710494449.094138, "uuid": "ad0f6f23baa54dcdb3bd118195525bed"}
2024-03-15T09:20:49 {"block_run_id": 21, "block_uuid": "load_location_dimension", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "Execute PipelineRun 3, BlockRun 21: pipeline export_to_big_query block load_location_dimension", "timestamp": 1710494449.202322, "uuid": "90c16f1e129d4c1ebd3f6fddd8c2c2d7"}
2024-03-15T09:20:49 {"block_run_id": 25, "block_uuid": "load_order_dimension", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "Execute PipelineRun 3, BlockRun 25: pipeline export_to_big_query block load_order_dimension", "timestamp": 1710494449.205547, "uuid": "56297fdd9f804396aa7bdef8c1853c58"}
2024-03-15T09:20:49 {"block_run_id": 29, "block_uuid": "load_shipping_dimension", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "Execute PipelineRun 3, BlockRun 29: pipeline export_to_big_query block load_shipping_dimension", "timestamp": 1710494449.248903, "uuid": "9ba2d84400ef402ead2a2ffc46aec475"}
2024-03-15T09:20:49 {"block_run_id": 27, "block_uuid": "load_product_dimension", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "Execute PipelineRun 3, BlockRun 27: pipeline export_to_big_query block load_product_dimension", "timestamp": 1710494449.328626, "uuid": "28babff778e046068bffed1bffd1689e"}
2024-03-15T09:20:56 {"block_run_id": 19, "block_uuid": "load_department_dimension", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "BlockRun 19 (block_uuid: load_department_dimension) completes.", "timestamp": 1710494456.857256, "uuid": "5a5ad86315b84b5da5c09149598a262d"}
2024-03-15T09:20:58 {"cpu": 0.55908203125, "cpu_total": 2, "cpu_usage": 0.279541015625, "memory": 3833.0, "memory_total": 7929.0, "memory_usage": 0.48341531088409634, "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "Pipeline export_to_big_query for run 3 in schedule 2 is alive.", "timestamp": 1710494458.191648, "uuid": "14cb006455d348e4be2abed2ef8f5b61"}
2024-03-15T09:20:58 {"block_run_id": 21, "block_uuid": "load_location_dimension", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "BlockRun 21 (block_uuid: load_location_dimension) completes.", "timestamp": 1710494458.381229, "uuid": "cab8251dcb744bdf9aab4cc9a6eb9756"}
2024-03-15T09:20:59 {"block_run_id": 24, "block_uuid": "load_metadata_dimensio", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "BlockRun 24 (block_uuid: load_metadata_dimensio) completes.", "timestamp": 1710494459.165351, "uuid": "f0608471b2e14c04994f5d44a03f2a60"}
2024-03-15T09:20:59 {"block_run_id": 29, "block_uuid": "load_shipping_dimension", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "BlockRun 29 (block_uuid: load_shipping_dimension) completes.", "timestamp": 1710494459.763007, "uuid": "75d57c7358f44a0088c5f0570d3ed88d"}
2024-03-15T09:20:59 {"block_run_id": 27, "block_uuid": "load_product_dimension", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "BlockRun 27 (block_uuid: load_product_dimension) completes.", "timestamp": 1710494459.87084, "uuid": "86a4b91199ab466aa29a7517019e3551"}
2024-03-15T09:21:00 {"block_run_id": 20, "block_uuid": "department_dim_to_big_query", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "Execute PipelineRun 3, BlockRun 20: pipeline export_to_big_query block department_dim_to_big_query", "timestamp": 1710494460.293495, "uuid": "70c72d5081cd4969b149bedefef1103c"}
2024-03-15T09:21:00 {"block_run_id": 17, "block_uuid": "load_customer_dimension", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "BlockRun 17 (block_uuid: load_customer_dimension) completes.", "timestamp": 1710494460.380438, "uuid": "c1032debc29040ddbcd3107e128868cd"}
2024-03-15T09:21:00 {"block_run_id": 22, "block_uuid": "location_dim_to_big_query", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "Execute PipelineRun 3, BlockRun 22: pipeline export_to_big_query block location_dim_to_big_query", "timestamp": 1710494460.393593, "uuid": "0d5501e245b2447c86ae830ea11123e1"}
2024-03-15T09:21:01 {"block_run_id": 25, "block_uuid": "load_order_dimension", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "BlockRun 25 (block_uuid: load_order_dimension) completes.", "timestamp": 1710494461.797628, "uuid": "13af47c8acf44a468e35669cc0a01d3f"}
2024-03-15T09:21:08 {"cpu": 0.5849609375, "cpu_total": 2, "cpu_usage": 0.29248046875, "memory": 3387.0, "memory_total": 7929.0, "memory_usage": 0.42716609912977677, "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "Pipeline export_to_big_query for run 3 in schedule 2 is alive.", "timestamp": 1710494468.050865, "uuid": "3950a85ad2cd41c1924d657533dc97ad"}
2024-03-15T09:21:09 {"block_run_id": 18, "block_uuid": "customer_dim_to_big_query", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "Execute PipelineRun 3, BlockRun 18: pipeline export_to_big_query block customer_dim_to_big_query", "timestamp": 1710494469.402957, "uuid": "57eafc072eea42a7809b661ce7065880"}
2024-03-15T09:21:09 {"block_run_id": 23, "block_uuid": "metadata_dim_to_big_query", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "Execute PipelineRun 3, BlockRun 23: pipeline export_to_big_query block metadata_dim_to_big_query", "timestamp": 1710494469.444437, "uuid": "0906017036f54b448e1d2194b4dce8ba"}
2024-03-15T09:21:09 {"block_run_id": 28, "block_uuid": "product_dim_to_big_query", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "Execute PipelineRun 3, BlockRun 28: pipeline export_to_big_query block product_dim_to_big_query", "timestamp": 1710494469.496572, "uuid": "67b893222ea34cafb42db243607d09e5"}
2024-03-15T09:21:09 {"block_run_id": 26, "block_uuid": "order_dim_to_big_query", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "Execute PipelineRun 3, BlockRun 26: pipeline export_to_big_query block order_dim_to_big_query", "timestamp": 1710494469.497726, "uuid": "020209e2cdf0493b98c70f22280d43ff"}
2024-03-15T09:21:09 {"block_run_id": 30, "block_uuid": "shipping_dim_to_big_query", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "Execute PipelineRun 3, BlockRun 30: pipeline export_to_big_query block shipping_dim_to_big_query", "timestamp": 1710494469.645952, "uuid": "2d41d7d861014228b80fc5156087a9db"}
2024-03-15T09:21:10 {"block_run_id": 20, "block_uuid": "department_dim_to_big_query", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "BlockRun 20 (block_uuid: department_dim_to_big_query) completes.", "timestamp": 1710494470.970599, "uuid": "9495d3a488e54585b014b6d3f78dfd9b"}
2024-03-15T09:21:13 {"block_run_id": 22, "block_uuid": "location_dim_to_big_query", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "BlockRun 22 (block_uuid: location_dim_to_big_query) completes.", "timestamp": 1710494473.892843, "uuid": "76b607b7f1f34e4a8321cea76a98fdda"}
2024-03-15T09:21:17 {"cpu": 0.6220703125, "cpu_total": 2, "cpu_usage": 0.31103515625, "memory": 4289.0, "memory_total": 7929.0, "memory_usage": 0.5409257157270778, "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "Pipeline export_to_big_query for run 3 in schedule 2 is alive.", "timestamp": 1710494477.707252, "uuid": "40ffc043a4d04b62a817746e050d4820"}
2024-03-15T09:21:18 {"block_run_id": 30, "block_uuid": "shipping_dim_to_big_query", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "EXCEPTION", "message": "BlockRun 30 (block_uuid: shipping_dim_to_big_query) failed.", "timestamp": 1710494478.815215, "uuid": "f7f7c53940a046ae802b7514df4ceb5f", "error": ["Traceback (most recent call last):\n  File \"/usr/local/lib/python3.10/site-packages/google/cloud/bigquery/client.py\", line 2491, in load_table_from_file\n    response = self._do_multipart_upload(\n  File \"/usr/local/lib/python3.10/site-packages/google/cloud/bigquery/client.py\", line 3046, in _do_multipart_upload\n    response = upload.transmit(\n  File \"/usr/local/lib/python3.10/site-packages/google/resumable_media/requests/upload.py\", line 153, in transmit\n    return _request_helpers.wait_and_retry(\n  File \"/usr/local/lib/python3.10/site-packages/google/resumable_media/requests/_request_helpers.py\", line 155, in wait_and_retry\n    response = func()\n  File \"/usr/local/lib/python3.10/site-packages/google/resumable_media/requests/upload.py\", line 149, in retriable_request\n    self._process_response(result)\n  File \"/usr/local/lib/python3.10/site-packages/google/resumable_media/_upload.py\", line 125, in _process_response\n    _helpers.require_status_code(response, (http.client.OK,), self._get_status_code)\n  File \"/usr/local/lib/python3.10/site-packages/google/resumable_media/_helpers.py\", line 108, in require_status_code\n    raise common.InvalidResponse(\ngoogle.resumable_media.common.InvalidResponse: ('Request failed with status code', 400, 'Expected one of', <HTTPStatus.OK: 200>)\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py\", line 613, in execute\n    result = __execute_with_retry()\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/shared/retry.py\", line 54, in retry_func\n    raise e\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/shared/retry.py\", line 38, in retry_func\n    return func(*args, **kwargs)\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py\", line 588, in __execute_with_retry\n    return self._execute(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py\", line 1077, in _execute\n    result = self.block.execute_sync(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1315, in execute_sync\n    raise err\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1224, in execute_sync\n    output = self.execute_block(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1530, in execute_block\n    outputs = self._execute_block(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1686, in _execute_block\n    outputs = self.execute_block_function(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1725, in execute_block_function\n    output = block_function_updated(*input_vars, **global_vars)\n  File \"<string>\", line 23, in export_data_to_big_query\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/io/bigquery.py\", line 319, in export\n    __process(database=database)\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/io/bigquery.py\", line 315, in __process\n    self.client.load_table_from_dataframe(df, table_id, job_config=config).result()\n  File \"/usr/local/lib/python3.10/site-packages/google/cloud/bigquery/client.py\", line 2727, in load_table_from_dataframe\n    return self.load_table_from_file(\n  File \"/usr/local/lib/python3.10/site-packages/google/cloud/bigquery/client.py\", line 2495, in load_table_from_file\n    raise exceptions.from_http_response(exc.response)\ngoogle.api_core.exceptions.BadRequest: 400 POST https://bigquery.googleapis.com/upload/bigquery/v2/projects/gothic-sylph-387906/jobs?uploadType=multipart: Invalid field name \"Days_for_shipping_(real)\". Fields must contain the allowed characters, and be at most 300 characters long. For allowed characters, please refer to https://cloud.google.com/bigquery/docs/schemas#column_names\n"], "error_stack": [["  File \"/usr/local/bin/mage\", line 8, in <module>\n    sys.exit(app())\n", "  File \"/usr/local/lib/python3.10/site-packages/typer/main.py\", line 311, in __call__\n    return get_command(self)(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/click/core.py\", line 1130, in __call__\n    return self.main(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/typer/core.py\", line 778, in main\n    return _main(\n", "  File \"/usr/local/lib/python3.10/site-packages/typer/core.py\", line 216, in _main\n    rv = self.invoke(ctx)\n", "  File \"/usr/local/lib/python3.10/site-packages/click/core.py\", line 1657, in invoke\n    return _process_result(sub_ctx.command.invoke(sub_ctx))\n", "  File \"/usr/local/lib/python3.10/site-packages/click/core.py\", line 1404, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n", "  File \"/usr/local/lib/python3.10/site-packages/click/core.py\", line 760, in invoke\n    return __callback(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/typer/main.py\", line 683, in wrapper\n    return callback(**use_params)  # type: ignore\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/cli/main.py\", line 163, in start\n    start_server(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/server/server.py\", line 743, in start_server\n    scheduler_manager.start_scheduler()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/server/scheduler_manager.py\", line 87, in start_scheduler\n    proc.start()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 121, in start\n    self._popen = self._Popen(self)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 224, in _Popen\n    return _default_context.get_context().Process._Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 281, in _Popen\n    return Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 19, in __init__\n    self._launch(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 71, in _launch\n    code = process_obj._bootstrap(parent_sentinel=child_r)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 314, in _bootstrap\n    self.run()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 108, in run\n    self._target(*self._args, **self._kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/db/process.py\", line 15, in start_session_and_run\n    results = target(*args)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/server/scheduler_manager.py\", line 50, in run_scheduler\n    LoopTimeTrigger().start()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/triggers/loop_time_trigger.py\", line 14, in start\n    self.run()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/triggers/time_trigger.py\", line 11, in run\n    schedule_all()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 1615, in schedule_all\n    PipelineScheduler(r).start()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/db/__init__.py\", line 157, in func_with_rollback\n    return func(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 190, in start\n    self.schedule()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/db/__init__.py\", line 157, in func_with_rollback\n    return func(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 318, in schedule\n    self.__schedule_blocks(block_runs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 602, in __schedule_blocks\n    job_manager.add_job(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/job_manager.py\", line 27, in add_job\n    self.queue.enqueue(job_id, target, *args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/queue/process_queue.py\", line 108, in enqueue\n    self.start_worker_pool()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/queue/process_queue.py\", line 183, in start_worker_pool\n    self.worker_pool_proc.start()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 121, in start\n    self._popen = self._Popen(self)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 224, in _Popen\n    return _default_context.get_context().Process._Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 281, in _Popen\n    return Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 19, in __init__\n    self._launch(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 71, in _launch\n    code = process_obj._bootstrap(parent_sentinel=child_r)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 314, in _bootstrap\n    self.run()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 108, in run\n    self._target(*self._args, **self._kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/queue/process_queue.py\", line 288, in poll_job_and_execute\n    worker.start()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 121, in start\n    self._popen = self._Popen(self)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 224, in _Popen\n    return _default_context.get_context().Process._Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 281, in _Popen\n    return Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 19, in __init__\n    self._launch(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 71, in _launch\n    code = process_obj._bootstrap(parent_sentinel=child_r)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 314, in _bootstrap\n    self.run()\n", "  File \"/usr/local/lib/python3.10/site-packages/newrelic/api/background_task.py\", line 117, in wrapper\n    return wrapped(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/queue/process_queue.py\", line 253, in run\n    start_session_and_run(args[1], *args[2], **args[3])\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/db/process.py\", line 15, in start_session_and_run\n    results = target(*args)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 1152, in run_block\n    return ExecutorFactory.get_block_executor(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py\", line 641, in execute\n    on_failure(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/db/__init__.py\", line 157, in func_with_rollback\n    return func(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 433, in on_block_failure\n    self.logger.exception(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/logging/logger.py\", line 30, in exception\n    self.__send_message('exception', message, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/logging/logger.py\", line 65, in __send_message\n    data['error_stack'] = traceback.format_stack(),\n"]], "error_stacktrace": ["400 POST https://bigquery.googleapis.com/upload/bigquery/v2/projects/gothic-sylph-387906/jobs?uploadType=multipart: Invalid field name \"Days_for_shipping_(real)\". Fields must contain the allowed characters, and be at most 300 characters long. For allowed characters, please refer to https://cloud.google.com/bigquery/docs/schemas#column_names"]}
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/google/cloud/bigquery/client.py", line 2491, in load_table_from_file
    response = self._do_multipart_upload(
  File "/usr/local/lib/python3.10/site-packages/google/cloud/bigquery/client.py", line 3046, in _do_multipart_upload
    response = upload.transmit(
  File "/usr/local/lib/python3.10/site-packages/google/resumable_media/requests/upload.py", line 153, in transmit
    return _request_helpers.wait_and_retry(
  File "/usr/local/lib/python3.10/site-packages/google/resumable_media/requests/_request_helpers.py", line 155, in wait_and_retry
    response = func()
  File "/usr/local/lib/python3.10/site-packages/google/resumable_media/requests/upload.py", line 149, in retriable_request
    self._process_response(result)
  File "/usr/local/lib/python3.10/site-packages/google/resumable_media/_upload.py", line 125, in _process_response
    _helpers.require_status_code(response, (http.client.OK,), self._get_status_code)
  File "/usr/local/lib/python3.10/site-packages/google/resumable_media/_helpers.py", line 108, in require_status_code
    raise common.InvalidResponse(
google.resumable_media.common.InvalidResponse: ('Request failed with status code', 400, 'Expected one of', <HTTPStatus.OK: 200>)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py", line 613, in execute
    result = __execute_with_retry()
  File "/usr/local/lib/python3.10/site-packages/mage_ai/shared/retry.py", line 54, in retry_func
    raise e
  File "/usr/local/lib/python3.10/site-packages/mage_ai/shared/retry.py", line 38, in retry_func
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py", line 588, in __execute_with_retry
    return self._execute(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py", line 1077, in _execute
    result = self.block.execute_sync(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1315, in execute_sync
    raise err
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1224, in execute_sync
    output = self.execute_block(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1530, in execute_block
    outputs = self._execute_block(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1686, in _execute_block
    outputs = self.execute_block_function(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1725, in execute_block_function
    output = block_function_updated(*input_vars, **global_vars)
  File "<string>", line 23, in export_data_to_big_query
  File "/usr/local/lib/python3.10/site-packages/mage_ai/io/bigquery.py", line 319, in export
    __process(database=database)
  File "/usr/local/lib/python3.10/site-packages/mage_ai/io/bigquery.py", line 315, in __process
    self.client.load_table_from_dataframe(df, table_id, job_config=config).result()
  File "/usr/local/lib/python3.10/site-packages/google/cloud/bigquery/client.py", line 2727, in load_table_from_dataframe
    return self.load_table_from_file(
  File "/usr/local/lib/python3.10/site-packages/google/cloud/bigquery/client.py", line 2495, in load_table_from_file
    raise exceptions.from_http_response(exc.response)
google.api_core.exceptions.BadRequest: 400 POST https://bigquery.googleapis.com/upload/bigquery/v2/projects/gothic-sylph-387906/jobs?uploadType=multipart: Invalid field name "Days_for_shipping_(real)". Fields must contain the allowed characters, and be at most 300 characters long. For allowed characters, please refer to https://cloud.google.com/bigquery/docs/schemas#column_names
2024-03-15T09:21:22 {"block_run_id": 28, "block_uuid": "product_dim_to_big_query", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "BlockRun 28 (block_uuid: product_dim_to_big_query) completes.", "timestamp": 1710494482.365818, "uuid": "b0b907768eba41dda375525592cc1fdf"}
2024-03-15T09:21:23 {"block_run_id": 23, "block_uuid": "metadata_dim_to_big_query", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "BlockRun 23 (block_uuid: metadata_dim_to_big_query) completes.", "timestamp": 1710494483.010501, "uuid": "235e8991f1c84a5d8f70658d49fc57c5"}
2024-03-15T09:21:23 {"block_run_id": 18, "block_uuid": "customer_dim_to_big_query", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "BlockRun 18 (block_uuid: customer_dim_to_big_query) completes.", "timestamp": 1710494483.899459, "uuid": "7898adbe892641dfb63b44ca82a35c0f"}
2024-03-15T09:21:27 {"block_run_id": 26, "block_uuid": "order_dim_to_big_query", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "BlockRun 26 (block_uuid: order_dim_to_big_query) completes.", "timestamp": 1710494487.759671, "uuid": "2b7b11781ca947589b1b6081f7b5375f"}
2024-03-15T09:21:28 {"cpu": 0.615234375, "cpu_total": 2, "cpu_usage": 0.3076171875, "memory": 2925.0, "memory_total": 7929.0, "memory_usage": 0.3688989784335982, "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "export_to_big_query", "hostname": "206c8e5e6755", "level": "INFO", "message": "Pipeline export_to_big_query for run 3 in schedule 2 is alive.", "timestamp": 1710494488.005899, "uuid": "f2b991b2dfc54fd2b5bdb7aaf3cd59dd"}
